test/mixer_test.py

```diff
@@ -86,7 +87,7 @@ def test_pre_init__zero_values(self):
         # default values. No way to check buffer size though.
         mixer.pre_init(22050, -8, 1)  # Non default values
         mixer.pre_init(0, 0, 0)  # Should reset to default values
-        mixer.init()
+        mixer.init(allowedchanges=0)
```

Comment:
How do we ensure that the original pre_init values are not used? By testing not equal.

```python
        self.assertNotEqual(mixer.get_init()[0], 22050)
        self.assertNotEqual(mixer.get_init()[1], -8)
        self.assertNotEqual(mixer.get_init()[2], 1)
```

But even that could result with weird hardware giving back those values as default. Imagine a 22050 only sound device.

Yeah, the `allowedchanges=False` is probably the best way to do this. But with our imaginary 22050 only device this test would mistakenly fail.

We'll see with wider testing, but I think this should be fine.

Reply:
That's a good point, but yes I think this will be fine.